deepchem/models/graph_models.py

```diff
@@ -754,15 +754,18 @@ def __init__(self,
         layers.GraphConv(layer_size, activation_fn=tf.nn.relu)
         for layer_size in graph_conv_layers
     ]
+    self.dense_layers = [
+        Dense(layer_size, activation = tf.nn.relu)
```

Comment:
Can you add a argument for `activation_fn` so the user can set their choice of activation function?